content/basics/basics.mdx

```diff
@@ -0,0 +1,327 @@
+---
+name: Basics
+route: /basics
+---
+
+import { Playground } from 'docz'
+
+# Basic Introduction
+
+[MobX](https://mobx.js.org/README.html) lets you create observable state. This
+library, `mobx-react` integrates it with React. It provides a way for you to
+create React components that respond to MobX state changes.
```

Comment:
The library isn't really creating anything, just wrapping.
```suggestion
enhance React components that respond to MobX state changes.
```

Reply:
I've rewritten it to "a way for you to make React components respond to MobX state changes now" now.